dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/GAENDataService.java

```diff
@@ -39,24 +39,26 @@
   /**
    * Returns the maximum id of the stored exposed entries for the given batch.
```

Comment:
Documentation (see also next comment): "the given batch" how does it relate to the parameters?